snowddl/converter/task.py

```diff
@@ -0,0 +1,92 @@
+from re import compile, DOTALL
+
+from snowddl.blueprint import ObjectType
+from snowddl.converter.abc_converter import ConvertResult
+from snowddl.converter.abc_schema_object_converter import AbstractSchemaObjectConverter
+from snowddl.converter._yaml import YamlLiteralStr
+from snowddl.parser.task import task_json_schema
+import json
+
+task_text_re = compile(r"^.*\n\)\sas(.*)$", DOTALL)
+after_syntax_re = compile(r"^(\w+)?\((.*)\)$")
+
+
+class TaskConverter(AbstractSchemaObjectConverter):
+    def get_object_type(self) -> ObjectType:
+        return ObjectType.TASK
+
+    def get_existing_objects_in_schema(self, schema: dict):
+        existing_objects = {}
+
+        cur = self.engine.execute_meta(
+            "SHOW TASKS IN SCHEMA {database:i}.{schema:i}",
+            {
+                "database": schema["database"],
+                "schema": schema["schema"],
+            },
+        )
+
+        for r in cur:
+            if r["state"] == "suspended":
+                continue
```

Comment:
Yes, let's remove the suspended check.

I think it should be easy for users to remove unwanted tasks manually after conversion.